src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: 60px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.link {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+  text-decoration: none;
+
+  margin: 10px;
+  padding: 20px 0;
+}
+
+.list > :last-child > .link {
+  margin-right: 0;
+}
```

Comment:
better to use margin-right = 20px than right and left 10px, it would be great to use construction where :not and :last-child used together, so it's looks like you add margin-right 20 px to all .link except last-child